pkg/kudoctl/packages/package.go

```diff
@@ -204,7 +203,7 @@ func (p *PackageFiles) getCRDs() (*PackageCRDs, error) {
 			APIVersion: apiVersion,
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			Name:   fmt.Sprintf("%s-%s", p.Operator.Name, rand.String(6)),
+			Name:   p.Operator.Name,
```

Comment:
I don't mind the determinism but by default, the instance will have the same name as Operator. I liked the fact, that instances were distinguishable from operator versions by name. Wdyt about:
```suggestion
			Name:    fmt.Sprintf("%s-instance", p.Operator.Name)
```

Reply:
@zen-dog could we do additional print columns in kubectl get instances to make the difference a little more?

Reply:
@zen-dog I did wonder about this and don't have any particularly strong preference, but I was comfortable with it having the same name as the Operator since it's clear from the API and what's being returned as to what object it is you're referring to.  As an outside contributor though I'm happy to defer to other's preferences.

Reply:
This change would break the `--instance` flag, because the use would get a different instances name then what was chosen (`%instanceName%-instance` instead of `%instanceName`).

Reply:
We could print extra columns but I prefer things being intrinsically clear e.g.:
```
Operator: zookeeper
OperatorVersion: zookeeper-0.2.0
Instance: zookeeper-0.2.0-jklsdm / zookeeper-0.2.0-instance
```
makes it immediately clear what the user is talking about.

Reply:
@zen-dog I've made the change as you've suggested.